azure-pipelines.yml

```diff
@@ -15,9 +15,7 @@ trigger:
   branches:
     include:
     - main
-pr:
-- main
-
+    
```

Comment:
I suggest we _want_ to validate PRs that change this file. what I did in my helix-machines work was
``` yaml

pr:
  autoCancel: true
  branches:
    include:
    - main
  paths:
    include:
    - azure-pipelines.yml
```